modules/moa-faq-installing.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/moa-faq.adoc
+
+
+[id="moa-faq-installing"]
+
+= Installation troubleshooting
+
+== Inspect install or uninstall logs
+
+Run the following command to display install logs. Replace <my_cluster_name> with the name of your cluster.
+
+[source,terminal]
+----
+$ moactl logs install <my_cluster_name>
+----
```

Comment:
General comment for all the procedures, but could most of them be written as a one-step unordered list? E.g.:

```
To display install logs:

* Run the following command, replacing `<my_cluster_name>` with the name of your cluster:
+
[source,terminal]
----
$ moactl logs install <my_cluster_name>
----

Reply:
Also in general suggest ending the sentences leading into the code blocks with `:`.